src/bindings/c/include/openvino/c/ov_common.h

```diff
@@ -167,10 +167,11 @@ typedef enum {
  * @ingroup ov_base_c_api
  * @brief This enum contains codes for element type.
  */
+
 typedef enum {
     UNDEFINED = 0U,  //!< Undefined element type
     DYNAMIC,         //!< Dynamic element type
-    BOOLEAN,         //!< boolean element type
+    BIN,             //!< boolean element type
```

Comment:
We cannot do it. It is not a backward compatible change

Reply:
Backward compatibility is a problem, but also there is BOOLEAN redefine issue if windows.h is referenced in the same C++/C files. Is there any good idea to solve both problem?